File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <  1 || height > 8);

    for (int i = 0; i < height; i++)
    {
        for (int n = height; n >= i; n--)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
             printf("#");
        }
        printf("  ");
        
        for (int y = 0; y <= i; y++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    # get input from user
    number = get_number()

    # print
    print_hashes(number)


def get_number():
    while True:
        number = get_int("Number: ")
        if number >= 1 and number <= 8:
            return number


def print_hashes(number_of_hashes):
    spaces = 1

    for i in range(number_of_hashes):
        print(" " * (number_of_hashes - spaces), end="")
        print("#" * spaces, end="")
        print("  ", end="")
        print("#" * spaces)
        spaces += 1


main()
